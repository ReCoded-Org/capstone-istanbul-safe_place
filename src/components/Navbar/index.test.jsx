import React from "react";
import renderer from "react-test-renderer";
import MainNavbar from "./index";

it("renders correctly and matches the snapshot", () => {
  const tree = renderer.create(<MainNavbar />).toJSON();
  expect(tree).toMatchSnapshot();
});