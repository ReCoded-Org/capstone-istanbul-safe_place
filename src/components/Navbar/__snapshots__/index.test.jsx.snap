// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly and matches the snapshot 1`] = `
<nav
  className="mainNavbar navbar navbar-expand-lg navbar-light"
>
  <div
    className="container"
  >
    <a
      className="navbar-brand"
      href="/"
    >
      <img
        alt="logo"
        src="Logo.svg"
      />
    </a>
    <button
      aria-controls="responsive-navbar-nav"
      aria-label="Toggle navigation"
      className="navbar-toggler collapsed"
      onClick={[Function]}
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      aria-expanded={null}
      className="navbar-collapse collapse"
      id="responsive-navbar-nav"
    >
      <div
        className="navbar-nav"
        onKeyDown={[Function]}
      >
        <a
          className="nav-link"
          data-rb-event-key="/about"
          href="/about"
          onClick={[Function]}
          onKeyDown={[Function]}
        >
          navbar.about
        </a>
        <a
          className="nav-link"
          data-rb-event-key="/blog"
          href="/blog"
          onClick={[Function]}
          onKeyDown={[Function]}
        >
          navbar.blog
        </a>
        <a
          className="nav-link"
          data-rb-event-key="/opportunities"
          href="/opportunities"
          onClick={[Function]}
          onKeyDown={[Function]}
        >
          navbar.opportunities
        </a>
        <a
          className="nav-link"
          data-rb-event-key="/contact"
          href="/contact"
          onClick={[Function]}
          onKeyDown={[Function]}
        >
          navbar.contact
        </a>
        <div
          className="dropdown dropdown"
          onKeyDown={[Function]}
        >
          <button
            aria-expanded={false}
            aria-haspopup={true}
            className="dropdown-toggle btn btn-none"
            disabled={false}
            id="dropdown-button"
            onClick={[Function]}
            type="button"
          >
            <img
              alt="Language icon"
              src="language.svg"
            />
          </button>
        </div>
        <button
          className="signInBtn"
          to="/signin"
          type="button"
        >
          navbar.buttons.signIn
        </button>
        <button
          className="signUpBtn"
          to="/signup"
          type="button"
        >
          navbar.buttons.signUp
        </button>
      </div>
    </div>
  </div>
</nav>
`;
